src/Microsoft.TemplateEngine.Abstractions/IPostAction.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.TemplateEngine.Abstractions
+{
+    public interface IPostAction
+    {
+        int Order { get; }
+
+        string Name { get; }
+
+        IReadOnlyList<IPostActionOperation> Operations { get; }
+
+        IReadOnlyList<IPostActionOperation> AlternateOperations { get; }
```

Comment:
Alternative is probably a better term than Alternate. The idea is that the Operations will run if they can be run. If they can, then the alternate operations will be ignored.
If the regular operations can't be run, then see if the alternate operations can be run (and run them if they can be).